src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -148,6 +146,8 @@ public ProNonCombatMoveAI(final ProAI ai) {
           break;
         }
       }
+    } else {
```

Comment:
PS: preferably update the commit message with some details so that your response is lasting.